pom.xml

```diff
@@ -122,6 +122,13 @@
 			<version>1.64</version>
 			<optional>true</optional>
 		</dependency>
+		
+		<!-- dependencies for Client Certificate Authentication -->
+		<dependency>
```

Comment:
Does this need any sort of open source approval?

Reply:
Yes, we will have to talk to crypto team for approval.